src/ontology/mondo-edit.obo

```diff
@@ -435509,6 +435509,18 @@ def: "A disease characteristic in which the disease has a known cause." [https:/
 is_a: MONDO:0700004 {source="https://github.com/monarch-initiative/mondo/issues/3177", source="https://orcid.org/0000-0002-4142-7153"} ! idiopathic vs non-idiopathic
 created_by: http://orcid.org/0000-0002-4142-7153
 
+[Term]
+id: MONDO:0700057
+name: neurological pain disorder
+def: "A nervous system disorder that has pain as a major feature." []
```

Comment:
need a dbxref for the definition

Reply:
done-added my ORCID (I hope that's an ok dbxref)